cmd/dep/govendor_importer.go

```diff
@@ -0,0 +1,219 @@
+		// 3. path and path prefix - any string with a slash (/) in it.
+		//   The path case could be a full path or just a prefix.
+		// Dep doesn't support build tags right now: https://github.com/golang/dep/issues/120
+		for _, i := range strings.Split(g.file.Ignore, " ") {
+			if !strings.Contains(i, "/") {
+				g.logger.Printf("Warning: Not able to convert ignoring of build tag '%v'", i)
+				continue
+			}
+			_, err := g.sm.DeduceProjectRoot(i)
+			if err == nil {
+				manifest.Ignored = append(manifest.Ignored, i)
+			} else {
+				g.logger.Printf("Warning: Not able to convert ignoring of build tag '%v'\n", i)
+			}
+		}
+	}
+
+	lock := &dep.Lock{}
+	for _, pkg := range g.file.Package {
+		// Path must not be empty
+		if pkg.Path == "" {
+			err := errors.New("Invalid govendor configuration, Path is required")
+			return nil, nil, err
+		}
+
+		// Obtain ProjectRoot. Required for avoiding sub-package imports.
+		// Use Path instead of Origin since we are trying to group by project here
+		pr, err := g.sm.DeduceProjectRoot(pkg.Path)
+		if err != nil {
+			return nil, nil, err
+		}
+		pkg.Path = string(pr)
+
+		// Check if it already existing in locked projects
+		if projectExistsInLock(lock, pkg.Path) {
+			continue
+		}
+
+		// Revision must not be empty
+		if pkg.Revision == "" {
+			err := errors.New("Invalid govendor configuration, Revision is required")
+			return nil, nil, err
+		}
+
+		if pkg.Version == "" {
+			// When no version is specified try to get the corresponding version
+			pi := gps.ProjectIdentifier{
+				ProjectRoot: gps.ProjectRoot(pkg.Path),
+			}
+			if pkg.Origin != "" {
+				pi.Source = pkg.Origin
+			}
+			revision := gps.Revision(pkg.Revision)
+			version, err := lookupVersionForLockedProject(pi, nil, revision, g.sm)
+			if err != nil {
+				// Only warn about the problem, it is not enough to warrant failing
+				g.logger.Println(err.Error())
+			} else {
+				pp := getProjectPropertiesFromVersion(version)
+				if pp.Constraint != nil {
+					pkg.Version = pp.Constraint.String()
+				}
+			}
+		}
+
+		// If there's a version, use it to create project constraint
+		pc, err := g.buildProjectConstraint(pkg)
+		if err != nil {
+			return nil, nil, err
+		}
+		manifest.Constraints[pc.Ident.ProjectRoot] = gps.ProjectProperties{Constraint: pc.Constraint}
+
+		lp := g.buildLockedProject(pkg, manifest)
+		lock.P = append(lock.P, lp)
+	}
+
+	return manifest, lock, nil
+}
+
+func (g *govendorImporter) buildProjectConstraint(pkg *govendorPackage) (pc gps.ProjectConstraint, err error) {
+	pc.Ident = gps.ProjectIdentifier{ProjectRoot: gps.ProjectRoot(pkg.Path), Source: pkg.Path}
+
+	if pkg.Version != "" {
+		pc.Constraint, err = g.sm.InferConstraint(pkg.Version, pc.Ident)
+		if err != nil {
+			return
+		}
+	} else {
+		pc.Constraint = gps.Any()
+	}
+
+	f := fb.NewConstraintFeedback(pc, fb.DepTypeImported)
+	f.LogFeedback(g.logger)
+
+	return
+}
+
+// buildLockedProject uses the package Rev and Comment to create lock project
```

Comment:
Looks like this comment is left over from using godep as a starting point and should be deleted?